documenting.rst

```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
++--------------+--------------------+------------------------------------------+
+| Russian (ru) |                    | `mail <https://mail.python.org/pipermail/|
+|              |                    | doc-sig/2019-May/004131.html>`__         |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Simplified   | `Shengjing         | `transifex <https://www.transifex.com/   |
+| Chinese      | Zhu <https://bugs. | python-doc/python-newest/language/       |
+| (zh-cn)      | python.org/user248 | zh_CN/>`__                               |
+|              | 11>`__             |                                          |
+|              |                    | `github <https://github.com/python/      |
+|              |                    | python-docs-zh-cn/>`__                   |
+|              |                    |                                          |
+|              |                    | `doc <https://docs.python.org/zh-cn/>`__ |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Spanish      | Raul Cumplido      | `github <https://github.com/raulcd/      |
+| (es)         |                    | python-docs.es/>`__                      |
+|              |                    |                                          |
+|              |                    | `old repo <https://github.com/PyAr/      |
+|              |                    | tutorial/>`__                            |
++--------------+--------------------+------------------------------------------+
+| Traditional  | 廖偉涵 Adrian Liaw | `github <https://github.com/python/      |
+| Chinese      |                    | python-docs-zh-tw/>`__                   |
+| (zh-tw)      |                    |                                          |
+|              |                    | `transifex <https://www.transifex.com/   |
+|              |                    | python-tw-doc/python-36-tw>`__           |
+|              |                    |                                          |
+|              |                    | `doc <https://docs.python.org/zh-tw/>`__ |
++--------------+--------------------+------------------------------------------+
+| Turkish (tr) |                    | `github <https://github.com/alaeddingurel|
+|              |                    | /python-docs-tr>`__                      |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+
+
+Starting a new translation
+--------------------------
+
+First subscribe to the `doc-sig
+<https://mail.python.org/mailman/listinfo/doc-sig>`_ mailing list,
+introduce yourself and the translation you're starting.
```

Comment:
```suggestion
and introduce yourself and the translation you're starting.
```

Reply:
Is it mandatory? I kind of prefer it without the and.

Reply:
Yes.  Or divide into two sentences. 'mailing list.  Next introduce'

Reply:
It just feels better with it IMO but whatever :)

Reply:
OK :) thanks